scheduler/pkg/store/mesh.go

```diff
@@ -407,6 +409,14 @@ func getInstanceCount(modelRuntimeInfo *pb.ModelRuntimeInfo) uint64 {
 	}
 }
 
+func (m *ModelVersion) ShouldScaleUp() bool {
+	if !util.AutoscalingEnabled(m.GetDeploymentSpec().GetMinReplicas(), m.GetDeploymentSpec().GetMaxReplicas()) {
```

Comment:
I am not sure if this condition is valid here as we want to decouple server scaling from model min/max replicas being defined.

Reply:
I've removed it, since the check is made in the agent for the model and the controller will check the server's params.